README.md

```diff
@@ -63,6 +63,13 @@ To build Prometheus from source code, You need:
 * NodeJS [version 16 or greater](https://nodejs.org/).
 * npm [version 7 or greater](https://www.npmjs.com/).
 
+Start by cloning the repository:
+
+```bash
+git clone https://github.com/prometheus/prometheus.git
+cd prometheus
+```
+
 You can directly use the `go` tool to download and install the `prometheus`
```

Comment:
The "download" part in this existing sentence is now a bit misleading, since the download / repo clone happens in the step before now. Maybe instead of "download and install" say "build and install" and remove the "directly" (since I think it refered to directly downloading & building without first cloning)?

Reply:
Oh yeah forgot to remove that, "build and install" sounds good.